src/flags.mli

```diff
@@ -775,6 +775,17 @@ module Interpreter : sig
   val steps : unit -> int
 end
 
+(** {2 ContractMonitor flags} *)
+module ContractMonitor : sig
+
+  (** Read input from file. *)
+  val input_file : unit -> string
+
+  (* (** Run number of steps, override the number of steps given in the input
+    file. *)
+  val steps : unit -> int *)
```

Comment:
Perhaps we could offer this option to the contract monitor as well. It should be straightforward, right?